project.clj

```diff
@@ -38,7 +38,7 @@
                                                                               cljsjs/create-react-class
                                                                               org.clojure/tools.logging
                                                                               net.cgrand/macrovich]]
-                                                   ^:source-dep [reagent "0.7.0"
+                                                   ^:source-dep [reagent "0.8.0"
```

Comment:
We have two parallel release branches, one for reagent 0.7.0 and one for 0.8.0. Will this technique work for both versions of Reagent?

Reply:
This can work for 0.7.0 because the Cljsjs libs also provide the old `cljsjs.*` names. But obviously that won't work with Shadow-CLJS. 

What is the reason for separate branch? Why not just use 0.8 always?

Reply:
If Re-frame-10x uses bundled Reagent 0.8, everything would still work if user has Reagent 0.7 on the project? User will have React 15 dependencies on the project, but Reagent 0.8 should work with those, except for new React 16 features.

Reply:
> What is the reason for separate branch? Why not just use 0.8 always?

Until recently we were using Reagent 0.7 while Reagent 0.8 was in alpha. However users were using Reagent 0.8/React 16 so we created a parallel release branch for them to be able to use re-frame-10x.